### ISSUE TITLE
Links to Docs consistently Broken 2.7.2 2.8.3....

### ISSUE THREAD

**Issue by gallexme** [ckan/ckan#4965]
### CKAN Version if known (or site URL)
2.7.2
2.8.3
and possibly a lot more Versions are affected.

### Please describe the expected behaviour
Click on Footers "CKAN-API" Link should redirect to the Documentation of that Version

### Please describe the actual behaviour
```
        \          SORRY            /
         \                         /
          \    This page does     /
           ]   not exist yet.    [    ,'|
           ]                     [   /  |
           ]___               ___[ ,'   |
           ]  ]\             /[  [ |:   |
           ]  ] \           / [  [ |:   |
           ]  ]  ]         [  [  [ |:   |
           ]  ]  ]__     __[  [  [ |:   |
           ]  ]  ] ]\ _ /[ [  [  [ |:   |
           ]  ]  ] ] (#) [ [  [  [ :===='
           ]  ]  ]_].nHn.[_[  [  [
           ]  ]  ]  HHHHH. [  [  [
           ]  ] /   `HH("N  \ [  [
           ]__]/     HHH  "  \[__[
           ]         NNN         [
           ]         N/"         [
           ]         N H         [
          /          N            \
         /           q,            \
        /                           \
```

### What steps can be taken to reproduce the issue? 
Install newest CKAN (2.8.3, or most other versions) click CKAN-API -> Get Dissapointed

**Comment by amercader** [comment]
We updated the docs to link consistently to the minor version a while ago,eg

https://docs.ckan.org/en/2.7/
https://docs.ckan.org/en/2.8/ (actually points to 2.8.2 due to #4892)

AFAICT we change to the API docs in the search page and the footer. (see eg https://beta.ckan.org/)

What links exactly and on what sites are you seeing broken @gallexme ?

**Comment by gallexme** [comment]
@amercader 
https://docs.ckan.org/en/ckan-2.7.2/api/ broken
https://docs.ckan.org/en/ckan-2.7.3/api/ works
https://docs.ckan.org/en/ckan-2.7.4/api/ broken
https://docs.ckan.org/en/ckan-2.7.5/api/ broken
https://docs.ckan.org/en/ckan-2.7.6/api/ broken

sure in 2.8.3 the footer links may now only go to 2.8 and not to 2.8.3
but back in 2.7.2 bootstrap2 they didnt they link to 2.7.2 not 2.7?
u just droppend the support for the footer API Docs links in most versions, without informing in changelog or so?

**Comment by gallexme** [comment]
here some example ckan sites from ur about/instances
http://data.zagreb.hr/dataset links to 2.5.2 not 2.5 (Docs still work tho, since they exist...)
https://data.kk.dk/dataset links to 2.5.4 not 2.5 (Docs still work tho, since they exist...)
http://offenedaten.frankfurt.de/dataset links to 2.7.2 not 2.7 (Docs dont  work!)

**Comment by amercader** [comment]
Tl;DR:

* Sites on CKAN >= 2.6: If the links are broken, the site needs upgrading to the latest patch release
* Sites on CKAN < 2.6: Links should work, as we kept the old versions on RTD for compatibility

@gallexme The links are fixed on all latest patch releases to point to the new two-digit URLs. So if a site is using the latest patch release (which they should be to get the latest fixes) it will show the correct links in the footer or search page (like beta.ckan.org does). The latest patch release is the only one the tech team supports.

If a site is using a custom theme it might have to update the links on their templates, and it is true that this could have been made clearer in the CHANGELOG (although the change in the URLs is listed there).

The individual patch version docs that you list were always broken, as we were changing the link on each release. Which was obviously bad and the reason we switched to the new much more stable and predictable setup. More info on the original issue: https://github.com/ckan/ckan/issues/4209

**Comment by amercader** [comment]
Hope this makes sense. If you think there's still an issue that we can solve feel free to reopen
